pkg/config/services.go

```diff
@@ -249,7 +249,7 @@ var SupportedServices = serviceConfigs{
 		},
 		DimensionRegexps: []*regexp.Regexp{
 			regexp.MustCompile("cluster/(?P<ClusterName>[^/]+)"),
-			regexp.MustCompile("service/(?P<ClusterName>[^/]+)/([^/]+)"),
+			regexp.MustCompile("service/(?P<ClusterName>[^/]+)/(?P<ServiceName>[^/]+)"),
```

Comment:
drive by: add `ServiceName` named capture group for extracting that dimension, and helping in the matching process with discerning with metrics corresponds to a cluster or to a service